src/components/Genomes/Browser/Popup/index.tsx

```diff
@@ -106,11 +108,15 @@ const formatData = (
       {
         name: 'E.C Number',
         Value:
-          attributes.ecnumber &&
+          attributes.ec_number &&
```

Comment:
One thing I have found elsewhere, is that there are differences between the assembly GFF attribute naming and the MAG GFF attribute naming (especially captialisation). That's why we have the `FORMAT.GENOME` and `FORMAT.ASSEMBLY_V5` things in `TrackColourPicker`. Do we need anything like that here?

Reply:
Actually no, although the name does imply that. I think these were refactored, to share the same popup code between assembly analysis contig viewer and genome browser viewer, at some point (and the name not changed I suppose). Not a problem unless assembly analyses also have EC Numbers in their GFF attrs, without the `_`.

Reply:
See https://github.com/EBI-Metagenomics/ebi-metagenomics-client/blob/bugfix/jira-EMG-5412-invalid-attributes-in-genome-gff/src/components/Analysis/ContigViewer/index.tsx#L31 for the relationship.